jobs/build/helm_sync/Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent any
+
+    parameters {
+        string(
+            name: "RPM_URL",
+            description: "Redistributable RPM URL. Example: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/helm/3.0.1/4.el7/x86_64/helm-redistributable-3.0.1-4.el7.x86_64.rpm",
+            defaultValue: ""
+        )
+        string(
+            name: "VERSION",
+            description: "Desired version name. Example: v3.0.1",
+            defaultValue: ""
+        )
+    }
+
+    stages {
+        stage("Download RPM") {
+            steps {
+                sh "wget ${params.RPM_URL} -O helm-redistributable.rpm"
+            }
+        }
+        stage("Extract RPM contents") {
+            steps {
+                sh "rpm2cpio helm-redistributable.rpm | cpio -id"
+                sh "rm -rf ${VERSION} && mkdir ${VERSION}"
```

Comment:
Created a first stage to validate params and fail in case they are not filled: https://github.com/openshift/aos-cd-jobs/pull/2105/commits/d359cb2ec13e32add5daa7211ad9fa212b502868

Regarding the `params.` prefix ... apparently it works without it (I haven't noticed the prefix omission) but it is confusing, better to keep everything explicit, so we know from where each var coming from: https://github.com/openshift/aos-cd-jobs/pull/2105/commits/7595cfc0f60d5915e6288bd53d593336b324fe0d